src/public/V8JsiRuntime.h

```diff
@@ -61,8 +54,27 @@ struct V8RuntimeArgs {
   size_t initial_heap_size_in_bytes{0};
   size_t maximum_heap_size_in_bytes{0};
 
-  bool enableGCApi{false};
-  bool ignoreUnhandledPromises{false};
+  // Padded to allow adding boolean flags without breaking the ABI
+  union {
+    struct {

```

Comment:
>struct { [](http://example.com/codeflow?start=4&length=8)

I do not think that the anonymous structs are the standard C++.

Reply:
I tried a few compilers in godbolt and they all like the anonymous structs. The only thing not working until c++20 is default initialization for bitfields in anonymous structs (so if we need any of these flags to default to something other than false, we'd need a constructor).

Reply:
Yes, in practice all most common compilers like MSVC, Clang, and GCC support it, but it is not part of the standard. E.g. see https://docs.microsoft.com/en-us/cpp/cpp/anonymous-class-types?view=msvc-160, https://stackoverflow.com/questions/16202576/c-anonymous-structs

Reply:
The MSVC fails to compile the anonymous structs if it is compiled with "Disable Language Extensions: Yes (/Za)".
I would suggest to name the struct field. E.g. `flags`.

Reply:
Makes sense, I'll do that. Thanks!